packages/terra-section-header/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-section-header",
+  "main": "lib/SectionHeader.js",
+  "version": "0.0.0",
+  "description": "Section Header presentational component that provides an onClick callback and animateable status icon.",
```

Comment:
I feel the description has too many implementation details. Also, I don't think section header should be included in the description. Perhaps we can get it to say something like this

`Interactive header component that provides a status indicator`

This could work as well too

`Header component that provides click functionality and a status indicator`

Reply:
Done c2a4558524abad60fbe8602a7c5fb3c75330854f

Reply:
I do not think the section header only acts as a status indicator. It can be used anywhere Modal Dialog Windows, Split Views, Popovers, Menus, Cards etc and it would mostly indicate the heading/title of the subcontent. Moreover the interactive part is optional.